### DISCUSSION TITLE
Больше возможностей!

### DISCUSSION THREAD

**Discussion by wedytd** [OpenVK/openvk#48] [Ideas] [answered]
**Расширить функционал админки**
верефикация , повысить до администатора , модератора , суппортки , разжаловать до пользователя , разбанить ,  и чтобы нельзя было забанить id1 , сделать шаблон пожаловаться уже...
![Screenshot_20210112-031657_Chrome](https://user-images.githubusercontent.com/55885089/104253141-d91e7400-5484-11eb-9a64-82cd5bf03ceb.png)

**Comment by WerySkok** [comment, marked as answer]
В целом давно уже сделано
